superset/migrations/env.py

```diff
@@ -54,6 +54,12 @@
 # ... etc.
 
 
+def print_duration(start_time: float) -> None:
+    duration = time.time() - start_time
+    formatted_time = time.strftime("%H:%M:%S", time.gmtime(duration))
+    logger.info(f"Migration scripts completed. Duration: {formatted_time}")
```

Comment:
Nitpick but usually you should let the logger format the message for you rather than doing it yourself, since the logger will do it lazily and avoid having to do the formatting if INFO leve llogs are off, for example.

Reply:
Done